install_files/ansible-base/group_vars/all/securedrop

```diff
@@ -50,4 +51,3 @@ appserver_dependencies:
   - apparmor-utils
   - redis-server
   - supervisor
-
```

Comment:
Aren't we supposed to keep the newlines at the end of the file for POSIX standard?